app/javascript/articles/Feed.jsx

```diff
@@ -33,7 +33,8 @@ export const Feed = ({ timeFrame, renderFeed }) => {
 
         // Remove that first one from the array.
```

Comment:
A couple of questions for my reviewers:

- Why do we, just a couple lines up (lines 28-31 in the code show below), "ensure first article is one with a main_image"? Why is it important that the first article contains a cover image?
```
        // Ensure first article is one with a main_image
        const featuredStory = feedItems.find(
          (story) => story.main_image !== null,
        );
```

- Why do we "remove that first one from the array"? Is it to separate the `featuredStory` from the `subStories` or is there another reason?

Reply:
Hey @juliannatetreault! Based on the fact that we are assigning that story we are assigning on those lines is saved to a const called featuredStory, the code tells me that the featured story actually IS the first article. And, since it is featured at the top of the feed, we want to always ensure that it has an image (the featured story appears at the top of the feed with a larger image than other stories/elements on the feed).

As for your second question, the reason we remove it is because we don’t want the featured story to appear again in the feed—if we didn’t remove it from the list of articles, then it would render twice which would be confusing to users :)

Reply:
Thanks for answering my questions, @vaidehijoshi 😊

Reply:
Anytime, @juliannatetreault! 😃

Reply:
It'd be AWESOME if this explanation found itself in the inline comments 🔥

Reply:
That's an excellent idea, @rhymes! I'll add an abridged explanation as a comment. 😄

Reply:
Great call to improve on this comment! I'm sure other folks would have had the same question when looking at this code, so this will be super helpful to our future selves 💯